### PR TITLE
Added 3 more icons (Kiwi Browser, Babbel and Life360)

### PR DESCRIPTION
**Pull request by JordyEGNL**
### Apps
[Kiwi Browser](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser) (`com.kiwibrowser.browser`)
[Babbel](https://play.google.com/store/apps/details?id=com.babbel.mobile.android.en) (`com.babbel.mobile.android.en`)
[Life360](https://play.google.com/store/apps/details?id=com.life360.android.safetymapd) (`com.life360.android.safetymapd`)

### Screenshots
image
image
image

### Comments
The Kiwi Browser is centered as a circle, but because the bottom part is missing, should I center it without the bottom part or leave it how it is?
image

### Fixed Issues
#264 #253

### CODE REVIEW

**Comment by JordyEGNL** [comment]
> Hello @JordyEGNL how do you take a screenshot before your icon is merged? Thanks in advance.

Download the develop branch and open it with Android Studio, then add your icons and changes, than you can run it on your device or an emulator.
image

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <icon drawable="@drawable/apple_music" package="com.apple.android.music" name="Apple Music" />
     <icon drawable="@drawable/aurora_store" package="com.aurora.store" name="Aurora Store" />
     <icon drawable="@drawable/authy" package="com.authy.authy" name="Authy" />
+    <icon drawable="@drawable/auxio" package="org.oxycblt.auxio" name="Auxio" />
```
Comment (JordyEGNL) [inline app/src/main/res/xml/grayscale_icon_map.xml:18]: For some reason there was a merge conflict, fixed it with this

**Comment by halacoglu** [comment]
> > Hello @JordyEGNL how do you take a screenshot before your icon is merged? Thanks in advance.
> 
> Download the develop branch and open it with Android Studio, then add your icons and changes, than you can run it on your device or an emulator. image

Thank you so much mate!

File: app/src/main/res/drawable/babbel.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <path
+      android:pathData="M6.63,10.66H9.88a1,1 0,0 1,1.16 1A1.24,1.24 0,0 1,9.86 13c-0.63,0 -1.27,0 -1.9,0 -0.29,0 -0.38,0.11 -0.39,0.36 0,0.52 -0.09,1 -0.12,1.56a1.4,1.4 0,0 1,-1 1.42,1.06 1.06,0 0,1 -1.35,-1 16.61,16.61 0,0 1,0.14 -1.72c0.07,-0.57 0.08,-0.58 -0.49,-0.58H3.17a1.06,1.06 0,0 1,-1.11 -0.69A1.28,1.28 0,0 1,2.42 11a1.34,1.34 0,0 1,0.9 -0.34Z"
+      android:fillColor="#000000"/>
+  <path
+      android:pathData="M6.55,9.92A1.51,1.51 0,0 0,8 8.39a1.06,1.06 0,0 0,-0.82 -1,1.27 1.27,0 0,0 -1.52,1c0,0.3 -0.08,0.61 -0.11,0.91l0,0.65Z"
+      android:fillColor="#000000"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/babbel.xml:15]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="m20.0462,11.5683a9.4824,9.4824 0,0 1,0.9222 0.5713,2.7465 2.7465,0 0,1 1.0024,2.0047 4.0095,4.0095 126.9195,0 1,-0.5914 2.6262,4.3001 4.3001,45 0,1 -2.8467,1.7441 7.0165,7.0165 0,0 1,-1.4434 0.1203c-1.3231,0 -2.6462,0 -3.9593,0a1.2229,1.2229 0,0 1,-1.263 -1.4735c0.1002,-0.7718 0.1504,-1.5637 0.2205,-2.3355l0.2706,-2.957c0.0601,-0.7117 0.1103,-1.4234 0.1804,-2.125 0.0902,-1.0926 0.1704,-2.1751 0.2907,-3.2677a1.3331,1.3331 45,0 1,1.263 -1.1126c1.3532,0 2.7164,0 4.0696,0A3.4582,3.4582 0,0 1,21.1488 6.9975,3.3379 3.3379,0 0,1 21.3794,9.764 3.0773,3.0773 0,0 1,20.0462 11.5683ZM16.3976,13.1921l-1.4835,0c-0.1704,0 -0.2606,0 -0.2706,0.2305 -0.01,0.2305 0,0.4611 -0.0601,0.6916 -0.0702,0.7117 -0.1303,1.4133 -0.2005,2.125 0,0.1804 0.0601,0.2406 0.2406,0.2406l0.842,0a20.3079,20.3079 135,0 0,2.3455 -0.0702,1.7341 1.7341,0 0,0 1.6739,-1.6238 1.253,1.253 0,0 0,-1.1828 -1.5136C17.6806,13.2121 17.0391,13.1921 16.4277,13.1921ZM16.4477,7.5688l-0.7618,0c-0.5713,0 -0.5713,0 -0.6014,0.5613l0,0.1904c-0.0601,0.7518 -0.1303,1.5035 -0.2005,2.2553 0,0.1904 0.0501,0.2706 0.2506,0.2606l2.0047,0A1.7241,1.7241 0,0 0,18.9236 9.4132,1.3432 1.3432,0 0,0 17.7709,7.6189 11.026,11.026 0,0 0,16.4778 7.5688Z"
      android:fillColor="#000000"/>
  <path
      android:pathData="m6.6446,10.6661l3.2577,0a1.0024,1.0024 0,0 1,1.1627 1.0024A1.2429,1.2429 0,0 1,9.8822 13.0117c-0.6315,0 -1.273,0 -1.9045,0 -0.2907,0 -0.3809,0.1103 -0.3909,0.3609 0,0.5212 -0.0902,1.0024 -0.1203,1.5637a1.4033,1.4033 45,0 1,-1.0024 1.4234,1.0625 1.0625,45 0,1 -1.3532,-1.0024 16.6493,16.6493 45,0 1,0.1403 -1.7241C5.3215,13.0618 5.3315,13.0518 4.7602,13.0518L3.1764,13.0518A1.0625,1.0625 45,0 1,2.0638 12.3601,1.283 1.283,135 0,1 2.4247,11.0069 1.3432,1.3432 45,0 1,3.3268 10.6661Z"
      android:fillColor="#000000"/>
  <path
      android:pathData="M6.5644,9.9244A1.5136,1.5136 135,0 0,8.0179 8.3908a1.0625,1.0625 0,0 0,-0.8219 -1.0024,1.273 1.273,0 0,0 -1.5236,1.0024C5.6723,8.6915 5.5921,9.0022 5.5621,9.3029l0,0.6515z"
      android:fillColor="#000000"/>
</vector>
```

File: svgs/babbel.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <path class="cls-1" d="M6.63,10.66H9.88a1,1,0,0,1,1.16,1A1.24,1.24,0,0,1,9.86,13c-.63,0-1.27,0-1.9,0-.29,0-.38.11-.39.36,0,.52-.09,1-.12,1.56a1.4,1.4,0,0,1-1,1.42,1.06,1.06,0,0,1-1.35-1,16.61,16.61,0,0,1,.14-1.72c.07-.57.08-.58-.49-.58H3.17a1.06,1.06,0,0,1-1.11-.69A1.28,1.28,0,0,1,2.42,11a1.34,1.34,0,0,1,.9-.34Z"/>
+      <path class="cls-1" d="M6.55,9.92A1.51,1.51,0,0,0,8,8.39a1.06,1.06,0,0,0-.82-1,1.27,1.27,0,0,0-1.52,1c0,.3-.08.61-.11.91l0,.65Z"/>
+    </g>
+  </g>
+</svg>
```
Comment (patrykmichalik) [inline svgs/babbel.svg:16]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg viewBox="0 0 24 24" version="1.1" id="svg15" xmlns="http://www.w3.org/2000/svg">
    <defs id="defs4">
        <style id="style2">
            .cls-1 {
            fill: #020202;
            }
        </style>
    </defs>
    <g id="g29" transform="matrix(1.0023638,0,0,1.0023638,-0.00105923,-0.01907096)"
        style="fill:#000000">
        <path
            d="m 20,11.56 a 9.46,9.46 0 0 1 0.92,0.57 2.74,2.74 0 0 1 1,2 4,4 0 0 1 -0.59,2.62 4.29,4.29 0 0 1 -2.84,1.74 7,7 0 0 1 -1.44,0.12 c -1.32,0 -2.64,0 -3.95,0 a 1.22,1.22 0 0 1 -1.26,-1.47 c 0.1,-0.77 0.15,-1.56 0.22,-2.33 l 0.27,-2.95 c 0.06,-0.71 0.11,-1.42 0.18,-2.12 0.09,-1.09 0.17,-2.17 0.29,-3.26 a 1.33,1.33 0 0 1 1.26,-1.11 c 1.35,0 2.71,0 4.06,0 A 3.45,3.45 0 0 1 21.1,7 3.33,3.33 0 0 1 21.33,9.76 3.07,3.07 0 0 1 20,11.56 Z m -3.64,1.62 h -1.48 c -0.17,0 -0.26,0 -0.27,0.23 -0.01,0.23 0,0.46 -0.06,0.69 -0.07,0.71 -0.13,1.41 -0.2,2.12 0,0.18 0.06,0.24 0.24,0.24 h 0.84 a 20.26,20.26 0 0 0 2.34,-0.07 1.73,1.73 0 0 0 1.67,-1.62 1.25,1.25 0 0 0 -1.18,-1.51 C 17.64,13.2 17,13.18 16.39,13.18 Z m 0.05,-5.61 h -0.76 c -0.57,0 -0.57,0 -0.6,0.56 v 0.19 c -0.06,0.75 -0.13,1.5 -0.2,2.25 0,0.19 0.05,0.27 0.25,0.26 h 2 A 1.72,1.72 0 0 0 18.88,9.41 1.34,1.34 0 0 0 17.73,7.62 11,11 0 0 0 16.44,7.57 Z"
            id="path6" style="fill:#000000" />
        <path class="cls-1"
            d="m 6.63,10.66 h 3.25 a 1,1 0 0 1 1.16,1 A 1.24,1.24 0 0 1 9.86,13 c -0.63,0 -1.27,0 -1.9,0 -0.29,0 -0.38,0.11 -0.39,0.36 0,0.52 -0.09,1 -0.12,1.56 a 1.4,1.4 0 0 1 -1,1.42 1.06,1.06 0 0 1 -1.35,-1 16.61,16.61 0 0 1 0.14,-1.72 C 5.31,13.05 5.32,13.04 4.75,13.04 H 3.17 A 1.06,1.06 0 0 1 2.06,12.35 1.28,1.28 0 0 1 2.42,11 1.34,1.34 0 0 1 3.32,10.66 Z"
            id="path8" style="fill:#000000" />
        <path class="cls-1"
            d="M 6.55,9.92 A 1.51,1.51 0 0 0 8,8.39 a 1.06,1.06 0 0 0 -0.82,-1 1.27,1.27 0 0 0 -1.52,1 C 5.66,8.69 5.58,9 5.55,9.3 v 0.65 z"
            id="path10" style="fill:#000000" />
    </g>
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
In the Babbel icon, the content seems to be slightly less than 20 px in width. Otherwise, everything looks good to me! I’d say the Kiwi Browser icon can stay the way it is.

**Comment by patrykmichalik** [comment]
Thanks, @JordyEGNL!

**Pull request merged**